src/profile_manager/templates/profile_manager/profile_list.html

```diff
@@ -9,7 +9,19 @@
 {% block head_title %}{% if title %}{{ title }}{% endif %}{% endblock %}
 
 {% block heading %}{{ config.custom_name_for_student }}s
-    {% if request.user.is_staff %}
+
+    <!-- block user list custom title and block object delete/edit buttons -->
+    {% if view_type == VIEW_TYPES.BLOCK %} in {% group_name %}: {{block_object.name}}
+    <a class="btn btn-warning" href="{% url 'courses:block_update' block_object.id %}" role="button" title="Edit this {% group_name %}">
+        <i class="fa fa-edit"></i>
+    </a>
+    <a class="btn btn-danger" href="{% url 'courses:block_delete' block_object.id %}" role="button" title="Delete this {% group_name %}">
+        <i class="fa fa-trash-o"></i>
+    </a>
+    {% endif %}
+    <!-- end of custom block user list header -->
+
+    {% if request.user.is_staff and view_type != VIEW_TYPES.BLOCK and block_object %}
```

Comment:
Why `and block_object`?  Seems irrelevant

Reply:
the thought was to catch requests that were made illegally via url editing by adding a conditional to accessing object data, but with the implementation of get_object_or_404 this becomes irrelevant, if it was ever relevant before, aha